docs/releasing/publishing.md

```diff
@@ -77,19 +77,25 @@ Developers should pair on releases. When remote working, it can be useful to be
   If you're following these instructions, you're probably doing a sequential release, meaning
   the tag should be 'latest'.
 
-  Enter `y` to continue. If you think the tag should be different, enter `N` to have the option to set your own npm tag.
+4. Enter `y` to continue. If you think the tag should be different, enter `N` to have the option to set your own npm tag.
 
-4. You will now be prompted to continue or cancel the release. Check the details and enter `y` to continue. If something does not look right, press `N` to cancel the release.
+5. You will now be prompted to continue or cancel the release. Check the details and enter `y` to continue. If something does not look right, press `N` to cancel the release.
+  
+  This step will also create a ZIP file containing the entirety of the release in the root of your local instance of 
+  govuk-frontend and will be referenced by the command line ("...and attach this").
```

Comment:
This sentence feels a bit cumbersome to me — maybe something more like this?

```suggestion
  This step will create a ZIP file containing the release in the root of your govuk-frontend git directory. You will need this file when creating the GitHub release.
```

Otherwise happy with this 👍